recipes/tree-gen/all/test_package/CMakeLists.txt

```diff
@@ -1,30 +1,23 @@
-cmake_minimum_required(VERSION 3.12)
-
-project(PackageTest CXX)
+cmake_minimum_required(VERSION 3.15)
+project(PackageTest LANGUAGES CXX)
 
 find_package(tree-gen REQUIRED CONFIG)
 
-# Generate the directory classes.
 generate_tree_py(
     "${CMAKE_CURRENT_SOURCE_DIR}/res/directory.tree"
-    "${CMAKE_CURRENT_SOURCE_DIR}/output/directory.actual.hpp"
```

Comment:
OK!

Reply:
The idea behind this `actual` and `golden` was that `test_package` could run `tree-gen`, then check that the output files (`*.actual`) were correct (equals to `*.golden`). But I guess we can skip those checks here.

Reply:
Yes, test package is not to validate project features, but the package only. Like: Are all artifacts installed? Can I link the generated library? Can I run the exported cmake macro? The test package should be very small.